File: Section Reports/Layout/SubReport/VB.NET/SubReport/rptNestedParent.vb
Public Class rptNestedParent
	Inherits GrapeCity.ActiveReports.SectionReport
	Dim _rpt As rptNestedChildMain
	Public Sub New()
		MyBase.New()
		InitializeComponent()
	End Sub
	Private Sub Detail_Format(ByVal sender As Object, ByVal e As EventArgs) Handles Detail.Format
		subOrders.Report = _rpt
	End Sub
	Private Sub rptNestedParent_ReportStart(ByVal sender As System.Object, ByVal e As EventArgs) Handles MyBase.ReportStart
		_rpt = New rptNestedChildMain()
		Dim dS As New Data.OleDBDataSource()
		dS.ConnectionString = My.Resources.ConnectionString
		dS.SQL = "Select * from employees"
		DataSource = dS
		
		ReportInfo1.FormatString = "Page {PageNumber} of {PageCount}"
		
		'
	End Sub
End Class

File: Section Reports/Layout/SubReport/C#/SubReport/rptNestedParent.cs
using System;
namespace GrapeCity.ActiveReports.Samples.SubReport
{
	/// <summary>
	/// Description of the outline of the rptNestedParent.
	/// 
	/// </summary>
	public partial class rptNestedParent : GrapeCity.ActiveReports.SectionReport
	{
		rptNestedChildMain rpt;
		public rptNestedParent()
		{
			//
			// ActiveReport designer support is required.
			//
			//
			InitializeComponent();
		}
		private void Detail_Format(object sender, EventArgs e)
		{
			subOrders.Report = rpt;
		}
		private void rptNestedParent_ReportStart(object sender, EventArgs e)
		{
			rpt = new rptNestedChildMain();
			Data.OleDBDataSource dS = new Data.OleDBDataSource();
			dS.ConnectionString = Properties.Resources.ConnectionString;
			dS.SQL = "Select * from employees";
			DataSource = dS;
			
			reportInfo1.FormatString = "Page {PageNumber} of {PageCount}";
			
			
		}
	}
}
